templates/workspaces/base/terraform/aad/outputs.tf

```diff
@@ -10,6 +10,10 @@ output "client_id" {
   value = azuread_application.workspace.application_id
 }
 
+output "scope_id" {
```

Comment:
Nit: could you add a comment here saying something like "This is where the scope_id is defined in case of automatic Workspace App Registrations"?

Reply:
I've added a comment in the main outputs.tf, not this one (as this whole file is conditional)